nautobot/docs/user-guide/administration/installation/install_system.md

```diff
@@ -40,406 +40,444 @@ You must select either MySQL (MariaDB is **not** supported) or PostgreSQL. Postg
 
-    If successful, you will enter a `mysql>` prompt. Type `status` to confirm your connection, or type `\q` to exit.
+            mysql> CREATE DATABASE nautobot;
+            Query OK, 1 row affected (0.00 sec)
 
-    !!! note
-        Replace `localhost` below with your database server if using a remote database.
+            mysql> CREATE USER 'nautobot'@'localhost' IDENTIFIED BY 'insecure_password';
+            Query OK, 0 rows affected (0.01 sec)
 
-    ```no-highlight
-    mysql --user nautobot --password --host localhost nautobot
-    ```
+            mysql> GRANT ALL ON nautobot.* TO 'nautobot'@'localhost';
+            Query OK, 0 rows affected (0.00 sec)
 
-    Example output:
+            mysql> \q
+            Bye
+            ```
 
-    ```no-highlight
-    Enter password:
-    Welcome to the MySQL monitor.  Commands end with ; or \g.
-    Your MySQL connection id is 13
-    Server version: 8.0.25-0ubuntu0.20.04.1 (Ubuntu)
-
-    Copyright (c) 2000, 2021, Oracle and/or its affiliates.
-
-    Oracle is a registered trademark of Oracle Corporation and/or its
-    affiliates. Other names may be trademarks of their respective
-    owners.
-
-    Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
-
-    mysql> status
-    --------------
-    mysql  Ver 8.0.25-0ubuntu0.20.04.1 for Linux on x86_64 ((Ubuntu))
-
-    Connection id:          13
-    Current database:       nautobot
-    Current user:           nautobot@localhost
-    SSL:                    Not in use
-    Current pager:          stdout
-    Using outfile:          ''
-    Using delimiter:        ;
-    Server version:         8.0.25-0ubuntu0.20.04.1 (Ubuntu)
-    Protocol version:       10
-    Connection:             Localhost via UNIX socket
-    Server characterset:    utf8mb4
-    Db     characterset:    utf8mb4
-    Client characterset:    utf8mb4
-    Conn.  characterset:    utf8mb4
-    UNIX socket:            /var/run/mysqld/mysqld.sock
-    Binary data as:         Hexadecimal
-    Uptime:                 26 min 31 sec
-
-    Threads: 2  Questions: 29  Slow queries: 0  Opens: 193  Flush tables: 3  Open tables: 112  Queries per second avg: 0.018
-    --------------
-
-    mysql> \q
-    Bye
-    ```
+        #### Verify MySQL Service Status
 
-=== "RHEL8 PostgreSQL"
+        You can verify that authentication works issuing the following command and providing the configured password.
 
-    ### Install PostgreSQL
+        If successful, you will enter a `mysql>` prompt. Type `status` to confirm your connection, or type `\q` to exit.
 
-    This will install the PostgreSQL database server and client.
+        !!! note
+            Replace `localhost` below with your database server if using a remote database.
 
-    ```no-highlight
-    sudo dnf install -y postgresql-server
-    ```
+        ```no-highlight
+        mysql --user nautobot --password --host localhost nautobot
```

Comment:
here too, user will be prompted for the password

Reply:
I'm happy with having the example show the password. The copy and paste of the command gets things in place. If we add anything with password on the line of the no-highlight, we lose the proper copy/paste capability.